content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+From Istio 1.7, it is possible to run Istiod on a separate, dedicated cluster (the "central control plane cluster") as shown in the diagram below. With this new deployment model, mesh operators can work purely on the central control plane cluster when there is a need to upgrade or re-configure Istio. Platform/service owners work solely on the data plane cluster to deploy their applications and apply any Istio resources or configurations. They don't have direct access to the central control plane cluster to deploy anything there, neither can the central control plane cluster process any Istio resources in that cluster. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
+
+{{< image width="100%"
+    link="single-cluster-central-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in a central control plane cluster"
+    >}}
+
+The diagram above shows a single cluster as the data plane for an Istio mesh. However, you can expand this deployment model to manage multiple Istio meshes from a central control plane cluster that runs multiple Istiod control planes:
+
+{{< image width="100%"
+    link="multiple-central-Istiods.svg"
+    alt="Istio meshes in single clusters with Istiod outside"
+    title="Multiple single clusters Istio meshes with Istiod outside"
+    caption="Multiple single clusters with multiple Istiod control planes in a central control plane cluster"
+    >}}
+
+Central control plane cluster can be used to host multiple Istiod control planes and each Istiod manages its own data plane clusters. In this model vendors could install another Istio mesh in the central control plane cluster and configure its `istio-ingress` gateway to route traffic from a data plane cluster to its corresponding Istiod control plane.
+
+You may further expand your data plane to multiple clusters, which are managed by the same Istiod running on the central control plane cluster. All of the data plane clusters will receive their configurations from the central control plane cluster per the diagram below.
+
+{{< image width="100%"
+    link="multiple-clusters-central-Istiod.svg"
+    title="Multicluster Istio mesh with Istiod outside"
+    caption="Multicluster Istio mesh with Istiod in a central control plane cluster"
+    >}}
+
+## Conclusion
+
+This new deployment model enables the Istio control plane to be run and managed by others who have the operational expertise in Istio.  This reduces the complexity and provides the best experience to platform/service owners by allowing them to focus on the core business logic of their applications and the configuration of their services.
```

Comment:
Here I would summarize benefits to the operator and the mesh admin (and again leave service owners out of it).

This really lets mesh operators be separate from mesh admins, allowing vendors to offer Istio control plane as a service, which was not really possible without this work. So for vendors this lets them provide a new offering, and for mesh admins it lets them use a managed service instead of operating the control plane themselves.